gruntfile.js

```diff
@@ -388,7 +280,7 @@ module.exports = function (grunt) {
       },
       build: {
         files: ['<%= src.js %>', '<%= src.css %>', '<%= src.tpl %>', '<%= src.html %>'],
-        tasks: ['build', 'shell:buildImage', 'shell:run', 'shell:cleanImages']
+        tasks: ['watchBuild:']
```

Comment:
I have not used Swarm and I was not confident enough to make any functional modification. According to you comment not, I removed lines 291-298. However, when doing so the task 'watchBuild' at line 59 (see above) is not required anymore. Shall I step back, remove that task, and leave a single 'watch:build:' task at line 283?